docs/millennium-os/manual/chapters/40_gcodes.md

```diff
@@ -500,13 +500,13 @@ G73 [Fnnn] [Rnnn] [Qnnn] [Xnnn] [Ynnn] [Znnn]
 G73 F500 R-5 Q1 Z-10 X10 Y10
 ```
 
-Run a peck drilling with partial retract cycle. **WARNING**: - this may not clear enough chips. You are likely better off using a drill cycle that retracts fully.
+Run a peck drilling with a partial retract cycle. **WARNING**: - this may not clear enough chips. You are likely better off using a drill cycle that retracts fully.
```

Comment:
This extra `a` reads wrong to me - the cycle name is 'Peck Drilling with Partial Retract' so 'Run a "peck drilling with partial retract" cycle' is how this is intended to be read.